src/harness/unittests/textChanges.ts

```diff
@@ -367,20 +367,22 @@ namespace M {
                 changeTracker.insertNodeAfter(sourceFile, findChild("M", sourceFile), createTestClass(), { prefix: newLineCharacter });
             });
         }
-        {

```

Comment:
I remove `{}` because function declared inside a statement or a block in strict mode. This is not cosmetic change.